addons/project/views/project_task_views.xml

```diff
@@ -646,7 +646,7 @@
                                     </div>
                                     <div class="oe_kanban_bottom_right" t-if="!selection_mode">
                                         <div  t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                            <field name="user_ids" widget="many2many_avatar_user" domain="[('share', '=', False)]"/>
+                                            <field name="user_ids" widget="many2many_avatar_user" context="{'active_test': True}" domain="[('share', '=', False)]"/>
```

Comment:
alternatively, it might be more clean to just remove the domain in XML: since the python field has domain `"[('share', '=', False), ('active', '=', True)]"`, the only reason we have the issue is that we override that domain with `domain="[('share', '=', False)]"`

```suggestion
                                            <field name="user_ids" widget="many2many_avatar_user"/>
```

Reply:
Thanks for the remark ! This has been changed accordingly :)